packages/adminrouter/extra/src/docs/api/nginx.agent.html

```diff
@@ -0,0 +1,683 @@
+    }
+
+    .route-type-file .route-type:hover {
+      box-shadow: 0 0 5px rgba(15, 106, 180, 1.0), inset 0 0 10px rgba(15, 106, 180, 1.0);
+    }
+
+    .route-type-lua .route-type:hover {
+      box-shadow: 0 0 5px rgba(107, 70, 102, 1.0), inset 0 0 10px rgba(107, 70, 102, 1.0);
+    }
+
+    .route-type-unknown .route-type:hover {
+      box-shadow: 0 0 5px rgba(66, 66, 66, 1.0), inset 0 0 10px rgba(66, 66, 66, 1.0);
+    }
+
+    .route-type-proxy .route-desc {
+      color: rgba(16, 165, 74, 1.0);
+    }
+
+    .route-type-redirect .route-desc {
+      color: rgba(197, 134, 43, 1.0);
+    }
+
+    .route-type-rewrite .route-desc {
+      color: rgba(234, 141, 6, 1.0);
+    }
+
+    .route-type-file .route-desc {
+      color: rgba(15, 106, 180, 1.0);
+    }
+
+    .route-type-lua .route-desc {
+      color: rgba(107, 70, 102, 1.0);
+    }
+
+    .route-type-unknown .route-desc {
+      color: rgba(66, 66, 66, 1.0);
+    }
+
+    .route-type-proxy .heading,
+    .route-type-proxy .route-meta {
+      border-color: rgba(16, 165, 74, 0.3);
+    }
+
+    .route-type-redirect .heading,
+    .route-type-redirect .route-meta {
+      border-color: rgba(197, 134, 43, 0.3);
+    }
+
+    .route-type-rewrite .heading,
+    .route-type-rewrite .route-meta {
+      border-color: rgba(234, 141, 6, 0.3);
+    }
+
+    .route-type-file .heading,
+    .route-type-file .route-meta {
+      border-color: rgba(15, 106, 180, 0.3);
+    }
+
+    .route-type-lua .heading,
+    .route-type-lua .route-meta {
+      border-color: rgba(107, 70, 102, 0.3);
+    }
+
+    .route-type-unknown .heading,
+    .route-type-unknown .route-meta {
+      border-color: rgba(66, 66, 66, 0.3);
+    }
+
+    .route-path {
+      display: inline-block;
+      padding: 0.5rem;
+      vertical-align: middle;
+    }
+
+    .route-desc {
+      right: 0;
+      display: block;
+      float: right;
+      padding: 0.5rem 1rem 0.5rem 0rem;
+      vertical-align: middle;
+      color: #aaa;
+    }
+
+    .route-meta {
+      padding: 0.5rem;
+      border: 1px solid #000;
+      border-top: 0;
+      border-radius: 3px;
+    }
+
+    .route-meta table {
+      width: 100%;
+    }
+
+    .route-meta table tr td:first-child {
+      white-space: nowrap;
+      width: 20%;
+    }
+
+    .arrow {
+      width: 0;
+      height: 0;
+      display: inline-block;
+      vertical-align: middle;
+    }
+
+    .arrow-down {
+      border-left: 5px solid transparent;
+      border-right: 5px solid transparent;
+      border-top: 5px solid black;
+      margin: 0 2.5px;
+    }
+
+    .arrow-right {
+      border-top: 5px solid transparent;
+      border-bottom: 5px solid transparent;
+      border-left: 5px solid black;
+      margin: 0 5px;
+    }
+  </style>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.0/jquery.min.js" integrity="sha256-JAW99MJVpJBGcbzEuXk4Az05s/XyDdBomFqNlM3ic+I=" crossorigin="anonymous"></script>
```

Comment:
That was actually in the standard examples on the jquery site! It's pretty neat.